skyvern/forge/sdk/routes/sdk.py

```diff
@@ -0,0 +1,182 @@
+import json
+
+import structlog
+from fastapi import Depends, HTTPException, status
+
+from skyvern import SkyvernPage
+from skyvern.core.script_generations.real_skyvern_page_ai import RealSkyvernPageAi
+from skyvern.forge import app
+from skyvern.forge.sdk.core import skyvern_context
+from skyvern.forge.sdk.core.skyvern_context import SkyvernContext
+from skyvern.forge.sdk.routes.routers import base_router
+from skyvern.forge.sdk.schemas.organizations import Organization
+from skyvern.forge.sdk.schemas.sdk_actions import (
+    ClickAction,
+    ExtractAction,
+    InputTextAction,
+    RunSdkActionRequest,
+    RunSdkActionResponse,
+    SelectOptionAction,
+)
+from skyvern.forge.sdk.services import org_auth_service
+from skyvern.forge.sdk.workflow.models.workflow import (
+    WorkflowRequestBody,
+    WorkflowRunStatus,
+)
+from skyvern.schemas.workflows import BlockType, WorkflowStatus
+
+LOG = structlog.get_logger()
+
+
+@base_router.post(
+    "/sdk/run_action",
```

Comment:
https://github.com/Skyvern-AI/skyvern/blob/83bd6f89b91c2c5c6f71946eacf4cc27a6985913/skyvern/forge/sdk/routes/run_blocks.py#L44

We talked abt making the API / SDK interfaces consistent to the UI (action block in this case).

We should make this endpoint `/run/action` from that perspective?